config/cookstyle.yml

```diff
@@ -1198,6 +1198,14 @@ Chef/Deprecations/HWRPWithoutUnifiedTrue:
   Include:
     - '**/libraries/*.rb'
 
+Chef/Deprecations/ResourceWithoutUnifiedTrue:
+  Description: Set `unified_mode true` in Chef Infra Client 15.3+ custom resources to ensure they work correctly in Chef Infra Client 18 (April 2022) when Unified Mode becomes the default.
+  StyleGuide: 'chef_deprecations_resourcewithoutunifiedtrue'
+  Enabled: true
+  VersionAdded: '7.12.0'
+  Include:
+    - '**/resources/*.rb'
```

Comment:
@tas50 this catches anything under `spec/resources` when it shouldn't be.

For example, in the ark cookbook, you will see the following:
```
spec/resources/default_spec.rb:1:1: R: Chef/Deprecations/ResourceWithoutUnifiedTrue: Set unified_mode true in Chef Infra Client 15.3+ custom resources to ensure they work correctly in Chef Infra Client 18 (April 2022) when Unified Mode becomes the default. (https://docs.chef.io/workstation/cookstyle/chef_deprecations_resourcewithoutunifiedtrue)
require 'spec_helper'
^
```

Reply:
Hmm got any ideas on how to fix that one?